EKF/common.h

```diff
@@ -367,7 +367,7 @@ struct parameters {
 	float EKFGSF_gyro_bias_gain{0.04f};	///< gain applied to integral of gyro correction for complementary filter (1/sec)
 	float EKFGSF_weight_min{0.0f};		///< minimum value of an individual model weighting
 	float EKFGSF_tas_default{15.0f};	///< default airspeed value assumed during fixed wing flight if no airspeed measurement available (m/s)
-	unsigned EKFGSF_reset_delay{2000000};	///< Number of uSec of bad innovations on main filter inpost takeoff phase before yaw is reset to EKF-GSF value
+	unsigned EKFGSF_reset_delay{1000000};	///< Number of uSec of bad innovations on main filter inpost takeoff phase before yaw is reset to EKF-GSF value
```

Comment:
Should be OK looking at replay data because this counter starts when GPS is rejected, which is already delayed from start of movement.